certified-connectors/DocuSignDemo/script.csx

```diff
@@ -536,6 +536,40 @@ public class Script : ScriptBase
     return body;
   }
   
+  private static JArray ParseContentAsJArray(string content, bool isRequest)
```

Comment:
Thanks for creating this method. This will be very helpful to work with JArrays.